app/TerminalViewController.m

```diff
@@ -104,7 +104,9 @@ - (void)viewDidLoad {
         [self.escapeKey setTitle:nil forState:UIControlStateNormal];
         [self.escapeKey setImage:[UIImage systemImageNamed:@"escape"] forState:UIControlStateNormal];
     }
-
+    [UserPreferences.shared observe:@[@"showStatusBar"] options:0 owner:self usingBlock:^(typeof(self) self) {
```

Comment:
This newline is fine

```suggestion

    [UserPreferences.shared observe:@[@"showStatusBar"] options:0 owner:self usingBlock:^(typeof(self) self) {
```

Reply:
This reminded me to change all of the observable names, that would have been a major oversight.